macs/m1.nix

```diff
@@ -0,0 +1,8 @@
+{
+  # 8C, 16 GB, 256 GB
```

Comment:
uber nit, feel free to ignore: What's the "256 GB"? Is that the size of the disk? Perhaps worth calling out explicitly? ("8 cores, 16 GB RAM, 256 GB storage")

Reply:
Yes, these are Macs. The reasoning is, that 256 GB RAM from Apple would bankrupt the foundation, so it cannot be that.